strawberryfields/engine.py

```diff
@@ -209,7 +218,7 @@ def _run(self, program, *, shots=1, compile_options={}, **kwargs):
             Result: results of the computation
         """
 
-        def _broadcast_nones(val, shots):
+        def _normalize_sample(val):
```

Comment:
I prefer `_broadcast_nones` in this context, as it is (a) slightly more explicit (the `None`'s are being broadcast a la NumPy terminology), and (b) 'normalize samples' has slightly different connotations in quantum physics.

But this is likely being removed in #111 anyway, so I would suggest changing it back just to avoid any annoying merge conflicts arising.

Reply:
That's fair -- I actually patched this as part of a commit from the `compile_in_stages` branch and just renamed it here. I'll revert the changes I made on top to avoid conflicts.